webapp/components/BuildListItem.jsx

```diff
@@ -34,8 +38,8 @@ export default class BuildListItem extends Component {
         ? `/${repo.full_name}/builds/${build.number}`
         : build.revision
         ? `/${repo.full_name}/revisions/${build.revision.sha}`
-        : this.context.router.location.pathname
```

Comment:
unrelated, but noticed this broken behavior when an endpoint wasnt returning correctly